src/main/java/io/debezium/connector/jdbc/dialect/postgres/EnumType.java

```diff
@@ -33,7 +33,7 @@ public String[] getRegistrationKeys() {
 
     @Override
     public String getTypeName(DatabaseDialect dialect, Schema schema, boolean key) {
-        LOGGER.warn("Cannot create enum types automatically, please create the table by hand. Using STRING fallback.");
+        LOGGER.trace("Cannot create enum types automatically, please create the table by hand. Using STRING fallback.");
```

Comment:
This change seems superfluous, was there a reason why this was necessary?

Reply:
not relevant to this pr, since in our case, source table have enum datatypes and since in production we keep log level info, this line is almost getting printed for every source record, our loki is full of these logs, i think trace is sufficient for this log line, what you think ?

Reply:
The concern I have here is that normally, you have INFO, WARN, and ERROR as the visible log levels, and if we move this to TRACE, then it could actually generate more questions from users who may otherwise know the issue if the WARN message was actually shown.

I'd prefer we leave this as `warn` and have users configure a specific logging category for the `EnumType` class to raise the level to ERROR or completely disable its logging entirely if the message is a problem for those who find it acceptable.

Reply:
yes agree, thanks, reverting